cmd/snap/cmd_booted.go

```diff
@@ -46,6 +47,11 @@ func (x *cmdBooted) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if release.OnClassic {
+		fmt.Fprintf(Stdout, "Ignoring 'booted' on classic")
```

Comment:
Thanks, yeah, we need to fix this. I created a trello card under "technical debt" to ensure it stays on the radar.